### PR TITLE
Added number.Local in combination with facebook searches

### PR DESCRIPTION
**Pull request by thehandsomezebra**
Facebook Pages (businesses, etc) in the US will not return any results with `RawLocal` formatting -- it requires `(000) 000-0000` Local formatting to get best results.

### CODE REVIEW

**Comment by sundowndev** [comment]
Hi, thanks for this PR!

Could you please update the tests as well ?

**Comment by thehandsomezebra** [comment]
> Hi, thanks for this PR!
> 
> Could you please update the tests as well ?

I think this should do it:
https://github.com/sundowndev/PhoneInfoga/pull/728/commits/f81430741e2e2100c826fd3a24b1c8906174af91

**Review by ankitk2601 — APPROVED** [review]

File: scanners/google_test.go [hunk 1]
```diff
@@ -246,8 +246,8 @@ func TestGoogleSearchScan(t *testing.T) {
 			expectedResult := []*GoogleSearchDork{
 				{
 					Number: "+33673421322",
-					Dork:   "site:facebook.com intext:\"33673421322\" OR intext:\"+33673421322\" OR intext:\"0673421322\"",
-					URL:    "https://www.google.com/search?q=site%3Afacebook.com+intext%3A%2233673421322%22+OR+intext%3A%22%2B33673421322%22+OR+intext%3A%220673421322%22",
+					Dork:   "site:facebook.com intext:\"33673421322\" OR intext:\"+33673421322\"  OR intext:\"06 73 42 13 22\" OR intext:\"0673421322\"",
```
Comment (sundowndev) [inline scanners/google_test.go:249]: There's a typo in your patch, you added an extra space before `OR intext:\"06 73 42 13 22\"`

**Review by sundowndev — CHANGES_REQUESTED** [review]
Unit tests still need to be fixed

**Comment by thehandsomezebra** [comment]
My apologies, I'm not well-versed in go and I'm having some difficulties with something so simple.  I will give it another attempt this weekend.

**Comment by sundowndev** [comment]
> My apologies, I'm not well-versed in go and I'm having some difficulties with something so simple. I will give it another attempt this weekend.

No problem, thank you for checking out anyway, I'll dig into this when I have some free time.

**Comment by sundowndev** [comment]
Closing for inactivity. Feel free to reopen and fix the PR.

**Pull request closed without merging**